index.html

```diff
@@ -6,121 +6,135 @@
+            margin-top: 3rem;
+            padding-top: 1rem;
+            border-top: 1px solid lightgrey;
+        }
+
+        #assignments > *:first-child {
+            border-top: none;
+        }
+    </style>
 </head>
 
 <body>
     <div class="container">
-        <div class="jumbotron">
-            <h1>JQuery Target Practice!</h1>
-        </div>
-
-        <div class="row well">
-            <h3>Instructions</h3>
-            <p>This page contains a bunch of html elements for you to play with.</p>
-            <p>Each
-                <code> section </code> has instructions for you to follow.</p>
-            <p>In order to complete all the exercises, you will need to refer to
-                <a href="https://api.jquery.com/">JQuery Documentation</a> and inspect the page</p>
+        <div class="jumbotron text-center">
+            <h1>JavaScript Target Practice!</h1>
         </div>
 
-        <div id="set-col-3" class="row well">
-            <div>
-                <h4>Use JQuery to select this div and add col-lg-3.</h4>
+        <div class="card">
+            <h2 class="card-header">Instructions</h2>
+            <div class="card-body">
+                <p>This page contains a bunch of html elements for you to play with.</p>
+                <p>Each
+                    <code> section </code> has instructions for you to follow.</p>
+                <p>In order to complete all the exercises, you will need to refer to
+                    <a href="https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector">
+                        documentation for selecting DOM Elements</a>.</p>
+                <p>Edit <code>targetPractice.js</code> to solve the problems.</p>
             </div>
         </div>
 
-        <div id="background-swap" class="row well">
-            <div class="col-lg-6" style="background: blue; color: white">
-                <h4>Change this background to be red</h4>
+        <div id="assignments">
+            <div id="set-col-3" class="mt-4">
+                <h3>Use JavaScript to select this div and add the class <code>col-3.</code></h3>
             </div>
-            <div class="col-lg-6" style="background: red; color: white">
-                <h4>Change this background to be blue</h4>
+
+            <div id="background-swap" class="row">
+                <div class="col-6 bg-primary text-white">
+                    <h3>Change this background to be red</h3>
+                </div>
+                <div class="col-6 bg-danger text-white">
+                    <h3>Change this background to be blue</h3>
+                </div>
             </div>
-        </div>
 
-        <div id="favorites" class="row well">
-            <h4>Add 3 of your favorite things as
-                <code>li</code> elements to the following
-                <code>ul</code>.
-            </h4>
-            <ul>
-                <li>Programming!</li>
-            </ul>
-        </div>
+            <div id="favorites">
```

Comment:
For this are you thinking `append` or `innerHTML`?

Reply:
This looks like the closest to jQuery... https://developer.mozilla.org/en-US/docs/Web/API/ParentNode/append

Reply:
I feel like we should discourage `innerHTML` from the start. I want it to be shown here, but we can do a small demonstration afterwards to demonstrate injecting some JS.

Right, I would think they should `appendChild` and element created from `document.createElement` and then use `innerText` to set the new `li`'s contents.

Think we should add links to that documentation in the description?

Reply:
I agree about avoiding innerHTML but I noticed in MDN this newer method `append` let's you append HTML strings instead of having to create the HTML element first. That's more modern and similar to jQuery. Thoughts on that?

Reply:
I might be using it wrong, but it seems that `append` is closer setting `innerText`. Testing it out on my machine, it just appends an escaped `<li>foo</li>` instead of interpreting it as HTML. That would kind of make sense to me. I wouldn't know how they would infer what should be escaped if we are just handing in a string. So it seems like the only safe thing to do is `createElement` and then `appendChild` it.